docs/fundamentals-fixing-a-memory-leak.md

```diff
@@ -130,7 +131,11 @@ Let's pretend that we don't know about this bad code. We just joined a new compa
 ╰→ com.example.leakcanary.MainActivity instance
 ```
 
-In words: the `FontsContract` class is a system class and has an `sContext` static field which references the `ExampleApplication` instance which has a `leakedViews` field which references an array list which references an array (the array backing the array list implementation) which contains a `TextView` which has an `mContext` field which references a destroyed instance of `MainActivity`. Let's highlight all references we are currently suspecting of causing this leak:
+Here's a description of that leak trace using words:
+
+> The `FontsContract` class is a system class (see `GC Root: System class`) and has an `sContext` static field which references an `ExampleApplication` instance which has a `leakedViews` field which references an `ArrayList` instance which references an array (the array backing the array list implementation) which has an element that references a `TextView` which has an `mContext` field which references a destroyed instance of `MainActivity`.
+
+LeakCanary highlights all references suspected of causing this leak using ~~~ underlines. Initially, all references are suspect:
```

Comment:
`are suspects`?

Reply:
I used suspect as an adjective instead of noun here. I think both work, not sure how to decide.